### DISCUSSION TITLE
Does prompt flow support integration with Phi-3-mini?

### DISCUSSION THREAD

**Discussion by niveditau** [microsoft/promptflow#3073] [General]
Does prompt flow support integration with Phi-3-mini? Any examples we can leverage? Thanks!

**Comment by brynn-code** [comment]
Hello, thanks for reaching us, we are working on the example to integrate with phi, this still working in progress and not merge yet. If you'd like to take a crack at it by yourselves, custom connection + custom python script nodes could be helpful. https://microsoft.github.io/promptflow/reference/tools-reference/python-tool.html
I'll update here once double confirmed the example status.
